README.md

```diff
@@ -36,7 +36,7 @@ When playing TGC, there are always cool "look what the AI did this time" moments
 
 Banks and Universities provide the player with new playstyles and methods to spend money. Universities are costly, but they provide prestige and a steady source of research points,  giving the player the opportunity of pulling ahead on research and becoming a technological powerhouse. Development banks will assist local landowners in developing their farms and considerably increasing farming capacity, allowing the player to invest more in RGO rather than just industrialisation.
```

Comment:
Suggestion;
to pull ahead on research and become a technological powerhouse. [...] allowing the player to develop their economy by investing in RGOs rather than just industrialization, creating a never-seen before playstyle.